### PR TITLE
add link to documentation  "Checking connection with Manager"

### LINKED ISSUE #2495: Add a link to "Check Connection" in the deploy_new_agents component
Hi team!

A new section will be created in the documentation to indicate how the connectivity between the agent and the manager is checked.
See the doc [section](https://documentation.wazuh.com/current/user-manual/agents/agent-connection.html)
It would be necessary to include a link to this new section in the `deploy_new_agent` component.

image

**Task:**
- [x] Create some mockups
- [x] Implement the mockup
- [x] Add the link

### PR DESCRIPTION
**Pull request by eze9252**
Hi team,  

This resolve: 

- Add link to documentation "Checking connection with Manager" when you add a new agent.

Captura de pantalla de 2021-04-06 16-17-25

How to test:

Agents > Deploy a new agent

Closes #2495

### CODE REVIEW

**Comment by Desvelao** [comment]
Add the change in `changelog.md` file.

**Comment by eze9252** [comment]
> Add the change in `changelog.md` file.

File: public/controllers/agent/components/register-agent.js [hunk 1]
```diff
@@ -473,6 +479,7 @@ export class RegisterAgent extends Component {
                   </EuiButton>
                 )}
               </EuiCopy>
+              <p style={pTextCheckConnectionStyle}>To check if the connection with the manager was successful, follow the <a href={urlCheckConnectionDocumentation} target="_blank">documentation.</a></p>
```
[the remark this answers is not in the document]
Comment (frankeros) [reply public/controllers/agent/components/register-agent.js:482]: also, the text could be a little more friendly
`To verify the connection with the Manager, please follow this document.`

File: public/controllers/agent/components/register-agent.js [hunk 2]
```diff
@@ -110,6 +110,10 @@ const sysButtons = [
   }
 ];
 
+const pTextCheckConnectionStyle = {
+  marginTop: '30px',
+};
```
Comment (gabiwassan) [inline public/controllers/agent/components/register-agent.js:115]: Could use `em` o `rem` units here? it's better for responsive renders.

**Review by gabiwassan — CHANGES_REQUESTED** [review]

**Review by pablomarga — APPROVED** [review]
LGTM!

**Pull request merged**